src/gazebo_mavlink_interface.cpp

```diff
@@ -487,7 +523,11 @@ void GazeboMavlinkInterface::OnUpdate(const common::UpdateInfo&  /*_info*/) {
 
   SendSensorMessages();
 
-  pollForMAVLinkMessages();
+  if (!serial_enabled_) {
```

Comment:
I'd rather invert the logic: `if (serial_enabled_) {`

Reply:
Fixed in c19fc4b.